packages/docs/src/components/docsearch/doc-search.tsx

```diff
@@ -1,5 +1,5 @@
 import type { SearchClient } from 'algoliasearch/lite';
-import { component$, useStore, useStyles$, useRef } from '@builder.io/qwik';
+import { component$, useStore, useStyles$, useRef, useBrowserVisibleTask$ } from '@builder.io/qwik';
```

Comment:
since we're already on this file: could we replace the deprecated `useRef` with `useSignal` als well? 👼